indico/modules/events/papers/client/js/components/RevisionReview.jsx

```diff
@@ -0,0 +1,134 @@
+              />
+            )}{' '}
+            <time dateTime={serializeDate(review.createdDt, moment.HTML5_FMT.DATETIME_LOCAL)}>
+              {serializeDate(review.createdDt, 'LL')}
+            </time>
+            {review.modifiedDt && (
+              <>
+                ·{' '}
+                <span
+                  className="review-comment-edited"
+                  title={serializeDate(review.modifiedDt, 'LL')}
+                >
+                  <Translate>edited</Translate>
+                </span>
+              </>
+            )}
+          </div>
+          <div className="review-group truncate-text">
+            <span title={review.group.title}>{review.group.title}</span>
+          </div>
+        </div>
+        <div
+          data-no-comment={!review.comment}
+          className={`i-timeline-item-box header-indicator-top ${
+            !review.comment ? 'header-only' : ''
+          }`}
+        >
+          <div className="i-box-header flexrow">
+            <div className="review-badges">
+              {review.proposedAction.name === 'accept' && (
+                <Translate>
+                  Proposed to <Param name="actionName" value="accept" wrapper={wrapper} />
+                </Translate>
+              )}
+              {review.proposedAction.name === 'reject' && (
+                <Translate>
+                  Proposed to <Param name="actionName" value="reject" wrapper={wrapper} />
+                </Translate>
+              )}
+              {review.proposedAction.name === 'to_be_corrected' && (
+                <Translate>
+                  Proposed to <Param name="actionName" value="correct" wrapper={wrapper} />
+                </Translate>
+              )}{' '}
+              {review.ratings.length > 0 &&
+                (review.score !== null && (
+                  <>
+                    <Translate>
+                      · score{' '}
+                      <Param
+                        name="prettyScore"
+                        value={review.score}
+                        wrapper={
+                          <span
+                            className="highlight bold semantic-text"
+                            title={Translate.string('Score: {score} / {maxScore}', {
+                              score: review.score,
+                              maxScore,
+                            })}
+                          />
+                        }
+                      />
+                    </Translate>{' '}
+                    {'('}
+                    <a className="js-ratings-toggle toggle-link">
+                      <span className="js-show-ratings">
+                        <Translate>show ratings</Translate>
+                      </span>
+                      <span className="weak-hidden js-hide-ratings">
+                        <Translate>hide ratings</Translate>
+                      </span>
+                    </a>
+                    {')'}
```

Comment:
actually, can't we just put a literal `)` here or will that result in an extra space?